app/Entirety/semantics/test_entities.json

```diff
@@ -0,0 +1,272 @@
+//this is the url to post this to: http://localhost:1026/v2/op/update
+//header:
+// fiware-service: your service used in the project
+// fiware-service-path:/
+// Content-Type: application/json
+//copy and paste this as payload
+
+{
+"actionType":"APPEND",
```

Comment:
Using lowercase will be better, i.e. "append"

Reply:
changed it. I only used this file to copy paste the contend into postman to have some test entities int the context broker. So, there is no deeper meaning to the content. should i delete the file?

Reply:
Perhaps such a demo file is something for the step by step tutorials to get started quickly and test all functionalities of Entirety? :D

Reply:
That's true. When the merge is finished, we should definitely provide more guidance.